tests/test_openqabot_simple.py

```diff
@@ -5,7 +5,7 @@
 import pytest
 import responses
 
-from openqabot.openqabot import OpenQABot
+from openqabot.openqabot import OpenQABot, QEM_DASHBOARD
```

Comment:
~~~
from openqabot.openqabot import OpenQABot
from openqabot.openqabot import QEM_DASHBOARD
~~~

Reply:
Why do we need this split? Is this something we can unify?

Reply:
done